lis/dataassim/obs/NASA_SMAPsm/read_NASASMAPsm.F90

```diff
@@ -734,12 +736,13 @@ subroutine read_SMAPL2sm_data(n, k,fname, smobs_inp, time)
   integer(hid_t)                 :: file_id
   integer(hid_t)                 :: dspace_id
   integer(hid_t)                 :: row_id, col_id
-  integer(hid_t)                 :: sm_gr_id,sm_field_id, sm_qa_id
+  integer(hid_t)                 :: sm_gr_id,sm_field_id, sm_qa_id, sm_surface_id
   integer(hid_t)                 :: sm_gr_id_A,sm_field_id_A
   integer(hid_t)                 :: vwc_field_id ! YK
   real,             allocatable  :: sm_field(:)
   real,             allocatable  :: vwc_field(:)! YK
   integer,          allocatable  :: sm_qa(:)
+  integer,          allocatable  :: sm_surface(:) ! ZH
```

Comment:
OK, but note that the name could be confusing, as if it is the actual surface SM field.

Reply:
I agree, this should be changed.